src-test/org/etools/j1939_84/controllers/part02/Part02Step02ControllerTest.java

```diff
@@ -51,7 +51,7 @@
 public class Part02Step02ControllerTest extends AbstractControllerTest {
     private static final int PART_NUMBER = 2;
     private static final int STEP_NUMBER = 2;
-    final int PGN = DM5DiagnosticReadinessPacket.PGN;
+    private static final int PGN = DM5DiagnosticReadinessPacket.PGN;
```

Comment:
You'd be better off to statically import DM5DiagnosticReadinessPacket.  Then you don't even need the member.